cpu/atmega_common/include/atmega_gpio.h

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (C) 2015 HAW Hamburg
+ *               2016 INRIA
+
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     cpu_atmega_common
+ * @ingroup     drivers_periph_gpio
+ * @{
+ *
+ * @file
+ * @brief       Macros and inline functions for accessing GPIOs of the ATmega family
+ *
+ * @author      René Herthel <rene-herthel@outlook.de>
+ * @author      Francisco Acosta <francisco.acosta@inria.fr>
+ * @author      Laurent Navet <laurent.navet@gmail.com>
+ */
+
+#ifndef ATMEGA_GPIO_H
+#define ATMEGA_GPIO_H
+#include <stdio.h>
+
+#include <avr/interrupt.h>
+
+#include "cpu.h"
+#include "board.h"
+#include "periph/gpio.h"
+#include "periph_conf.h"
+#include "periph_cpu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define GPIO_BASE_PORT_A        (0x20)
```

Comment:
Those macros as well:
```suggestion
#define ATMEGA_GPIO_BASE_PORT_A       (0x20)
```

Reply:
Done